shivanand.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+BASE_URL = "https://reqres.in/api/users"
+
+def apiCall():
+    response = requests.get(BASE_URL)
+    print(response.json())
```

Comment:
A function should return a value, not print the value. Change it so that your method is returning a value

Reply:
changed into return